lib/AST/PlatformKind.cpp

```diff
@@ -75,6 +77,9 @@ static bool isPlatformActiveForTarget(PlatformKind Platform,
     case PlatformKind::iOS:
     case PlatformKind::iOSApplicationExtension:
       return Target.isiOS() && !Target.isTvOS();
+    case PlatformKind::macCatalyst:
+    case PlatformKind::macCatalystApplicationExtension:
+      return tripleIsMacCatalystEnvironment(Target) && !Target.isTvOS();
```

Comment:
Should the `!isTvOS()` check be in `tripleIsMacCatalystEnvironment()` instead of in this particular call site?

Reply:
Done.